sdk/cognitiveservices/Vision.ContentModerator/tests/Helpers/Utilities.cs

```diff
@@ -653,9 +652,9 @@ public static bool VerifyJob(Content content, Job j)
                 TestBase.ErrorMessage = $"{content.GetDescription()} job : " + (string.IsNullOrEmpty(j.Id) ? "" : j.Id) + error;
                 return string.IsNullOrEmpty(error);
             }
-            catch (Exception e)
+            catch (Exception)
```

Comment:
It concerns me in how many places this antipattern exists. Perhaps we need a best practices section of our guidelines that warn against stuff like this with reasons.

Reply:
It is also weird to see many instances of a `try/catch` that catch and just rethrow.